packages/webchat/src/core/api.tsx

```diff
@@ -135,6 +122,7 @@ export default class WebchatApi {
     }
   }
 
+  // TODO: Fix this
   async deleteMessages(conversationId: uuid) {
     try {
       await this.axios.post(
```

Comment:
There's a API route already to delete all messages of a conversation. We could add a way to do this by socket as well.